### ISSUE TITLE
Update changelog

### ISSUE THREAD

**Issue by cthoyt** [scolby33/OCSPdash#21]
1. It’s almost a year out of date (lol)
2. Exsize it from the readme and put in its own file

**Comment by scolby33** [comment]
Agreed.

My least favorite part is replicating it from the root of the repo to the docs.

**Comment by cthoyt** [comment]
Good point. RST has a way to import and embed RST documents though, so it might be possible to have it single source (I tried this once and sort of had it working)

**Comment by scolby33** [comment]
From the repo to the docs isn't too bad since you control sphinx, can you extract only one subheading?

From the docs to the repo is hard because I don't think github uses a fully-featured RST parser.
